gpus.go

```diff
@@ -35,11 +36,14 @@ func GPUsGetMetrics() *GPUsMetrics {
 	return ParseGPUsMetrics()
 }
 
-func ParseAllocatedGPUs() float64 {
-	var num_gpus = 0.0
-
+func getSacctData() []byte {
 	args := []string{"-a", "-X", "--format=Allocgres", "--state=RUNNING", "--noheader", "--parsable2"}
```

Comment:
Note that in SLURM 20.11+ `Allocgres` is deprecated, consider changing to `AllocTRES` like in #34, or to make it version independent fist check if `sacct --helpformat` returns `AllocGRES` as a possible value.

Reply:
I am aware of that. However, since I don't have a slurm v20 cluster, I'm not risking changing the code myself to make it compatible with that version. Instead, I'm adding tests for the current code which should make it easier to add support for slurm 20 **later** (e.g. in another PR).